data_import/utils/csv_meta.py

```diff
@@ -22,7 +22,8 @@ class CsvMeta(object):
         'first_name',
         'title',
         'department',
-        'salary',
+        'base_salary',
```

Comment:
Great catch that this needs to be updated! It reminded me that I actually need to remove the calculated salary column from the ETL pipeline. https://github.com/datamade/bga-payroll/pull/361